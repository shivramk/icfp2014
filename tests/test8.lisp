(define (main x) (* x x))
