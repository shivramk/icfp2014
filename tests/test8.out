LD 0 0
LD 0 0
MUL
